### PR TITLE
Ensure that genesis-time is not used for jailing and slashing

### PR DESCRIPTION
**Pull request by haasted**
This is a fix for #19.

### CODE REVIEW

File: x/slashing/keeper/abci_test.go [hunk 1]
```diff
@@ -196,6 +196,88 @@ func TestBeginBlocker(t *testing.T) {
...
+		batch.Write()
+	}
+
+	// Validator 2 must remain bonded.
+	validator, found := stakingKeeper.GetValidatorByConsAddr(ctx, sdk.GetConsAddress(pk2))
+	require.True(t, found)
+	require.Equal(t, stakingtypes.Bonded, validator.GetStatus())
+
+	// create a third block 25 hours later and ensure jailing
+	ctx = ctx.WithBlockHeight(3).WithBlockTime(now.Add(25 * time.Hour))
```
Comment (blewater) [inline x/slashing/keeper/abci_test.go:265]: With 61 minutes the validator is jailed.

**Review by blewater — COMMENTED** [review]
Good find and test.
The test with 61 minutes later still finds val2 jailed.

File: x/slashing/keeper/abci_test.go (hunk 1 above)
Comment (haasted) [reply x/slashing/keeper/abci_test.go:265]: Ah, yes. I see that the default jailing time is actually only 60 minutes. Different value (24h) in the emoney-3 genesis.

**Review by blewater — APPROVED** [review]
Great job identifying the issue and fixing it.

**Pull request merged**